### PR TITLE
Add the first draft of permission plugin documentation

### PR DESCRIPTION
**Pull request by LittleBaiBai**
This is a new feature for TAP 1.6

### CODE REVIEW

File: tap-gui/auth.hbs.md [hunk 1]
```diff
@@ -112,6 +112,8 @@ auth:
   allowGuestAccess: true
 ```
 
+**Note** Guests cannot view catalog entities if you have permission plugin enabled.
```
Comment (andreizimin) [inline tap-gui/auth.hbs.md:115]: The only part that needs to stay here. With a link to the doc set-up-tap-gui-permissions-framework.hbs.md

**Comment by RichardJJG** [comment]
@LittleBaiBai, would you say this is ready for merging now?

**Comment by RichardJJG** [comment]
Hi, @andreizimin, can you speak to this by any chance? I'm always keen to merge PRs as quickly as possible. 

@LittleBaiBai, do you happen to go by a different name in VMware email and Slack? I'm not finding anything for "Yuxin Bai".

**Comment by RichardJJG** [comment]
Closing at @andreizimin's request

**Pull request closed without merging**